cal/static/jsx/query.jsx

```diff
@@ -19,6 +19,7 @@ var RSearch = React.createClass({
 
 var RReset = React.createClass({
     onclick: function() {
+        $("input[name='user'").prop('checked', true);
```

Comment:
Huh. Didn't know you could do that in jquery. What does passing in `true` do? Does it toggle, or set to `true`?